Makefile

```diff
@@ -191,9 +186,7 @@ v8:
 	tools/make-v8.sh
 	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
 
-test: all
-	$(MAKE) build-addons
-	$(MAKE) build-addons-napi
+test: build-addons
```

Comment:
This does the configure + build dance before running any of the JS tests.  Probably should be moved but I kept it as close as possible to the existing order for now.